pages/common/nasm.md

```diff
@@ -0,0 +1,23 @@
+# nasm
+
+> The Netwide Assembler, a portable 80x86 assembler.
+
+- Assemble raw binary file `source` from `source.asm`:
+
+`nasm {{source.asm}}`
+
+- Assemble binary file `output_file` of the specified format:
```

Comment:
In the first example, the format of the binary file assembled is format raw. I'll make this explicitly clear.